app/js/services/addon.js

```diff
@@ -140,3 +140,58 @@ AddonService.uninstall = function(origin) {
     }).catch(reject);
   });
 };
+
+AddonService.getMerger = function() {
+  return new Promise((resolve, reject) => {
+    var name = window.prompt('Enter a name for this add-on', 'Merged Addon ' + new Date().toISOString());
+    if (!name) {
+      reject();
+      return;
+    }
+
+    var merger = new AddonMerger(name);
+    merger.manifest.customizations = [{
+      filter: window.location.origin,
+      scripts: ['main.js']
+    }];
+
+    resolve(merger);
+  });
+};
+
+AddonService.merge = function(origins) {
```

Comment:
We shouldn't be using ```origin``` to uniquely identify add-ons. We should use ```manifestURL```. I think using ```origin``` is ok in this case though, since we're only handling Customizer add-ons.